### PR TITLE
docs(core): how to run locally

### PR DESCRIPTION
**Pull request by dominiksipowicz**
The purpose of this PR is to clarify how to run the project locally and highlight the fact that the default provider is just a stripped-down version of a real-life integration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ Demo live at: [demo.vercel.store](https://demo.vercel.store/)
 - Kibo Commerce Demo: https://kibocommerce.vercel.store/
 - Commerce.js Demo: https://commercejs.vercel.store/
 
+## Run minimal version locally 
+
+> deafult local provider `@vercel/commerce-local` has disabled all features (cart, auth) and use static files for the backend
```
Comment (okbel) [inline README.md:22]: deafult typo.

> To run a minimal version of Next.js Commerce you can start with the default local provider...

File: README.md [hunk 2]
```diff
@@ -92,7 +101,7 @@ Follow our docs for [Adding a new Commerce Provider](packages/commerce/new-provi
 
 If you succeeded building a provider, submit a PR with a valid demo and we'll review it asap.
 
-## Contribute
+## Contribute / add new provider
```
Comment (okbel) [inline README.md:104]: No need

File: README.md (hunk 1 above)
Comment (okbel) [reply README.md:22]: Install all dependencies. (There's an issue that pops up - described in the troubleshoot section) Could be useful to point that out.

**Review by okbel — CHANGES_REQUESTED** [review]

**Comment by FranciscoMoretti** [comment]
I just stumbled upon this problem. Thank you for fixing the docs! 👍

**Review by okbel — APPROVED** [review]

**Pull request merged**